.github/workflows/deploy.yml

```
@@ -14,11 +14,37 @@ env:
   AWS_REGION: us-west-2
   ECR_REPOSITORY: fixfox-api
   IMAGE_TAG: ${{ github.sha }}
+  NODE_VERSION: '20'
 
 jobs:
+  # Testing temporarily disabled
+  # test:
+  #   name: Run Tests
+  #   runs-on: ubuntu-latest
+  #   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)